apps/portlet-documents/src/main/webapp/vue-app/attachment/extensions.js

```diff
@@ -54,30 +54,35 @@ export function installExtensions() {
     },
   };
 
-  extensionRegistry.registerExtension('activity', 'action', Object.assign({
-    id: 'download',
-    labelKey: 'documents.label.download',
-    isEnabled: activity => {
-      if (activity.templateParams) {
-        const docPaths = activity.templateParams.DOCPATH && activity.templateParams.DOCPATH.split('|@|')
-                          || (activity.templateParams.nodePath && [activity.templateParams.nodePath]);
-        return docPaths && docPaths.length === 1;
-      }
-    },
-    rank: 0,
-  }, downloadHandlerExtension));
+  Vue.prototype.$transferRulesService.getTransfertRulesDownloadDocumentStatus()
```

Comment:
IMO, it's better to make the method `getTransfertRulesDownloadDocumentStatus` returns a Promise that returns `true` or `false` and not a JSON Object.

Reply:
@ahmadmido Could you take care of this feedback